tutorials/nservicebus-101/lesson-2/Solution/ClientUI/Program.cs

```diff
@@ -39,6 +37,8 @@ await endpointInstance.Stop()
 
         static async Task RunLoop(IEndpointInstance endpointInstance)
         {
+            var logger = LogManager.GetLogger<Program>();
```

Comment:
@DavidBoike this goes against our advice that loggers should be static. People should never do this in a production system